docs/blueprints/EP022.rst

```diff
@@ -0,0 +1,256 @@
+:EP: 22
+:Title: Kytos Pathfinder Filter Paths by Metadata
+:Authors:
+    - David Ramirez <drami102@fiu.edu>
+    - Marvin Torres <mtorr068@fiu.edu>
+:Created: 2020-10-05
+:Kytos-Version: 2020.2b
+:Status: Draft
+
+Abstract
+========
+
+This blueprint is to add support for filtering by metadata using Kytos Pathfinder's `v2/` endpoint. In particular,
```

Comment:
After the senior project, the endpoint was `v2/best-constrained-paths`. It is easy for me to change it to that. But I feel that dedicating a unique endpoint in general will require other NApps to distinguish endpoints and thus increase coupling overall.

As a result, I feel it may be best to keep it as 'v2/' and future iterations of this endpoint.

Reply:
I would suggest to not mention the v2, because this is a internal convention for endpoint versions. You are talking about the root endpoint "POST /". Yes, you can mention the version if you want, but we need to make clear here that his is about the root endpoint.